ctre/__init__.py

```diff
@@ -14,7 +14,6 @@
     BaseTalonPIDSetConfigUtil,
     BaseTalonPIDSetConfiguration,
     BufferedTrajectoryPointStream,
-    CANBusAddressable,
```

Comment:
Can you comment these out instead of deleting them if you're not going to remove the compiled wrapper? I'd like to continue using create-imports.

Reply:
I'll just leave these here; I don't really care enough.